tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -528,6 +539,7 @@ def param_value(self, verbose, name, ptype, value, textfile, binfile, command):
                                             executable_command = ' '.join([sys.executable, f])
                             if executable_command:
                                 command = command.replace(cl[0], executable_command, 1)
+                    command = split_with_single_quote_enclosed_fixup(command)
```

Comment:
Why should this only be applied after the long `if` block? Couldn't the same also apply to the first item of `command` and therefore affect the `if` block?